inst/HTM0/HTM0_whyUseMizer.Rmd

```diff
@@ -77,108 +79,112 @@ We can do the same with Biomass to see when, if any, of the species collapse. Fo
 
 # select the species  (or should we set this up with all species, like plotlyBiomass, should we make these plots in built functions?)
```

Comment:
It would be good to have plot functions in mizer for comparing the result of different scenarios. Is plotting the ratio the best choice though? For example when the values are small in both scenarios, then a small numerical error can lead to a large value of the ratio. And what if a value is zero in one scenario. This can lead to an infinite ratio.

Reply:
I'll dig out what I did for my talk in Hobart because back then I believe I had thought at bit about how to compare two different size spectra that works better than taking the ratio.

Reply:
Cool, let's see what you dig up.

Reply:
I am afraid I did not dig up anything particularly interesting. What I had thought about was actually not how to compare total biomasses as a function of time, but how to compare abundance densities as a function of size. There, instead of the ratio
```
finalN(sim0) / finalN(sim)
```
I have used the relative difference
```
(finalN(sim) - finalN(sim0)) / (finalN(sim) + finalN(sim0)) 
```
which produces
![newplot (2)](https://user-images.githubusercontent.com/1340713/132937332-00989f63-03f6-4ea1-9251-0c82435c33a8.png)
This may be a bit better because it no longer blows up but always lies between -1 and 1, but it still emphasises differences in areas where the abundance is small, that a fisher probably wouldn't care about.

Reply:
I like this, I'll change it in my code before making the next pull request.